### ISSUE TITLE
import/order complains about inline requires

### ISSUE THREAD

**Issue by mAAdhaTTah** [benmosher/eslint-plugin-import#1936]
Rule config:

```js
    'import/order': [
      'error',
      {
        groups: ['builtin', 'external', 'parent', 'sibling', 'index'],
        'newlines-between': 'never',
      },
    ],
```

Code:

```js
import React from 'react';
import PropTypes from 'prop-types';
import { createGlobalStyle, ThemeProvider } from 'styled-components';

export const GlobalStyle = createGlobalStyle`
  @font-face {
    font-display: swap;
    font-family: 'font-name';
    src: url(${require('../fonts/font-name.eot')});
    src: url(${require('../fonts/font-name.woff2')}) format('woff2'),
      url(${require('../fonts/font-name.woff')}) format('woff'),
      url(${require('../fonts/font-name.ttf')}) format('truetype');
  }
`
```

It complains about the newline after the `styled-components` import as well as a newline after the last inline require. Not 100% sure this is a bug, admittedly, but I thought it would excuse inline requires that aren't assigned based on [this](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#pass). Happy to just disable this rule on this file if needed.

**Comment by ljharb** [comment]
Although all of those requires should be non-inline, i agree it's a bug that the rule considers them.

**Comment by golopot** [comment]
I noticed another false positive:

```js
const foo = [
  require('fs'),
  require('./a'),
]
```

In this case elements order is likely intentional. The rule should not complain about it.

**Comment by ljharb** [comment]
given that it’s inline, i think that alone means the rule shouldn’t complain.
